app/main/lib/similarity.py

```diff
@@ -64,7 +64,7 @@ def get_body_for_text_document(params, mode):
       params['content'] = None
 
     if mode == 'store':
-      allow_list = set(['language', 'content', 'created_at', 'models', 'context', 'callback_url'])
+      allow_list = set(['language', 'content', 'created_at', 'models', 'context', 'callback_url', 'content_hash'])
```

Comment:
should doc_id be on the allow list?

Reply:
I agree this is a bit wonky missing it here, but in all branches I'm looking at where its in store mode, the doc_id is handled separately.